src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/UnaryLogicalOp.java

```diff
@@ -4,5 +4,5 @@
 
 public interface UnaryLogicalOp extends LogicalOperator
 {
-	
+    String toString();
```

Comment:
Shouldn't this method declaration already be in the `LogicalOperator` super-interface?

In fact, I wonder why you need to declare these methods at all. Every class in Java implicitly extends `java.Object` and, thus, has a `toString()`method anyways, and this method may be overridden in subclasses.